content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -17,6 +17,11 @@ a *kubeconfig file*. This is a generic way of referring to configuration files.
 It does not mean that there is a file named `kubeconfig`.
 {{< /note >}}
 
+{{< warning >}}
+Using unverified kubeconfig files could result in malicious code execution and file exposure. 
+Inspect your kubeconfig files carefully and only use config files from trusted sources.
```

Comment:
I love this, but I worry it may scare users in a way they don't have enough information to fix on their own?
Here's one idea for how to soften it a bit:
```suggestion
Kubeconfig files have powerful features that can run commands when they are used. Using a specially-crafted kubeconfig file could result in malicious code execution or file exposure. 
Only use kubeconfig files from trusted sources. If you must use an untrusted kubeconfig file (or create a kubeconfig file from untrusted input), inspect or filter it carefully beforehand, much as you would a shell script or other program.
```